clubs/models.py

```diff
@@ -71,6 +61,104 @@ class Club(models.Model):
         null=True
     )
 
+    occupation = models.TextField(
```

Comment:
Ok sounds good! The only reason I bring this up is because if we input this data through the admin interface, by default it will show up as a text area and not an input text field.